packages/CodingStandard/src/Fixer/ControlStructure/ShortIfReturnFixer.php

```diff
@@ -0,0 +1,216 @@
+				$tokenId++;
+			}
+			$tokens->clearAt($tokenId);
+			$tokens->clearEmptyTokens();
+			$tokens->insertAt($index, $shortReturnTokens);
+
+		}
+	}
+
+	/**
+	 * Returns the name of the fixer.
+	 *
+	 * The name must be all lowercase and without any spaces.
+	 *
+	 * @return string The name of the fixer
+	 */
+	public function getName()
+	{
+		return self::class;
+	}
+
+	/**
+	 * Returns the priority of the fixer.
+	 *
+	 * The default priority is 0 and higher priorities are executed first.
+	 *
+	 * @return int
+	 */
+	public function getPriority()
+	{
+		return 0;
+	}
+
+	/**
+	 * Returns true if the file is supported by this fixer.
+	 *
+	 * @param \SplFileInfo $file
+	 *
+	 * @return bool true if the file is supported by this fixer, false otherwise
+	 */
+	public function supports(\SplFileInfo $file)
+	{
+		return true;
+	}
+
+	private function getConditionEnd(Tokens $tokens, $conditionStart)
+	{
+		$nextTokenId = $conditionStart;
+		$startingBracesCount = 1;
+		while ($startingBracesCount > 0) {
+			$nextTokenId = $tokens->getNextMeaningfulToken($nextTokenId);
+			$nextToken = $tokens[$nextTokenId];
+			if ($nextToken->getContent() === '(') {
+				$startingBracesCount++;
+			} elseif ($nextToken->getContent() === ')') {
+				$startingBracesCount--;
+			}
+		}
+		$conditionEnd = $tokens->getPrevMeaningfulToken($nextTokenId);
+		return $conditionEnd;
+	}
+
+	private function checkConditionBody(Tokens $tokens, $tokenId)
+	{
+		if ($tokens[$tokenId]->getContent() !== '{') {
+			return -1;
+		}
+		$tokenId = $tokens->getNextNonWhitespace($tokenId);
+
+		if (!$tokens[$tokenId]->isGivenKind(T_RETURN)) {
+			return -1;
+		}
+		$tokenId = $tokens->getNextNonWhitespace($tokenId);
+		if (strtolower($tokens[$tokenId]->getContent()) !== 'true') {
+			return -1;
+		}
+		$tokenId = $tokens->getNextNonWhitespace($tokenId);
+		if ($tokens[$tokenId]->getContent() !== ";") {
+			return -1;
+		}
+		$tokenId = $tokens->getNextNonWhitespace($tokenId);
+		if ($tokens[$tokenId]->getContent() !== '}') {
+			return -1;
+		}
+		return $tokenId;
+	}
+
+	private function checkStatementAfterCondition(Tokens $tokens, $tokenId)
+	{
+		$tokenId = $tokens->getNextNonWhitespace($tokenId);
+		if (!$tokens[$tokenId]->isGivenKind(T_RETURN)) {
+			return -1;
+		}
+		$tokenId = $tokens->getNextNonWhitespace($tokenId);
+		if (strtolower($tokens[$tokenId]->getContent()) !== 'false') {
+			return -1;
+		}
+		$tokenId = $tokens->getNextNonWhitespace($tokenId);
+		if ($tokens[$tokenId]->getContent() !== ";") {
+			return -1;
+		}
+		return $tokenId;
+	}
+
+	private function createShortReturnTokens(Tokens $tokens, $conditionStart, $conditionEnd)
+	{
+		$shortReturnTokens = [
+			new Token([T_RETURN, 'return']),
+			new Token([T_WHITESPACE, ' ']),
+		];
+		$tokenId = $tokens->getNonEmptySibling($conditionStart, 1);
+		while ($tokenId !== $conditionEnd) {
+			$shortReturnTokens[] = $tokens[$tokenId];
+			$tokenId = $tokens->getNonEmptySibling($tokenId, 1);
+		}
```

Comment:
What exactly does this method? Maybe extract to new one with descriptive name would make it easier to understand